services/src/datasets/postgres.rs

```diff
@@ -548,6 +549,22 @@ where
         Ok(DatasetIdAndName { id, name })
     }
 
+    async fn update_dataset_symbology(
+        &self,
+        dataset: DatasetId,
+        symbology: &Symbology,
+    ) -> Result<()> {
+        let conn = self.conn_pool.get().await?;
+
+        let stmt = conn
+            .prepare("UPDATE datasets SET symbology = $2 WHERE id = $1;")
+            .await?;
+
+        conn.execute(&stmt, &[&dataset, &symbology]).await?;
```

Comment:
same here

by the way, why do we have this method twice?

Reply:
d5a4af89d4158c1e587be6e14db88fafd70ec890

we have it twice because of pro and non pro, where only pro needs to check permissions.